cli/src/cli.spec.ts

```diff
@@ -182,6 +184,18 @@ describe('CLI Integration Tests', () => {
         });
     });
 
+    test('example server command - starts server and responds to requests', async () => {
+        const serverCommand = 'calm server -p 3001 --schemaDirectory ../../dist/calm/';
+        const serverProcess = exec(serverCommand);
+        // Give the server some time to start
+        await new Promise(resolve => setTimeout(resolve, 10000));
```

Comment:
```suggestion
        await new Promise(resolve => setTimeout(resolve, 10 * millisPerSecond));
```

Is this timeout required? I only ask we await the npx command from returning in the call to `exec` - does the express server really take another 10s to start up?

Reply:
Updated to 5 seconds